ceph.spec.in

```diff
@@ -885,69 +887,38 @@ rm -rf $RPM_BUILD_ROOT
 %endif
 
 %pre base
-%if 0%{?_with_systemd}
-  %if 0%{?suse_version}
-    # service_add_pre and friends don't work with parameterized systemd service
-    # instances, only with single services or targets, so we always pass
-    # ceph.target to these macros
-    %service_add_pre ceph.target
-  %endif
+%if 0%{?suse_version}
+%service_add_pre ceph.target
 %endif
 
 %post base
-/sbin/ldconfig
```

Comment:
The Fedora Packaging Guidelines say "every binary RPM package that stores shared library files in any of the dynamic linker's default paths, must call ldconfig in `%post` and `%postun`"

I just looked in the `%files` sections and it appears that only `ceph-base` satisfies this condition. The packages `ceph-mds`, `ceph-mon`, `ceph-osd,` or `ceph-radosgw` do not appear to contain any shared library-type files.

I will reinstate `/sbin/ldconfig` in the `ceph-base` package, but it doesn't need to be in any of the others.